internal/bminventory/inventory.go

```diff
@@ -2560,6 +2568,19 @@ func (b *bareMetalInventory) GetHost(_ context.Context, params installer.GetHost
 	return installer.NewGetHostOK().WithPayload(&host.Host)
 }
 
+func (b *bareMetalInventory) GetHostById(hostId string) (*common.Host, error) {
```

Comment:
in my opinion `internal/host` is a better place for this function

Reply:
why?